bot.js

```diff
@@ -1,197 +1,91 @@
 require("dotenv").config();
 
 let TelegramBot = require("node-telegram-bot-api");
+let game = require("./game.js");
 
 if (!process.env.TELEGRAM_TOKEN) {
   throw new Error("TELEGRAM_TOKEN env variable is missing");
 }
 
 const bot = new TelegramBot(process.env.TELEGRAM_TOKEN, { polling: true });
-let gameStart = false;
-let spentCities = new Set();
-let lastLetter = "";
 
-bot.on("polling_error", m => console.log(m));
+let session = [];
+let usedKeyWords = ["/start", "начать", "сдаюсь"];
```

Comment:
You can store your commands in specific object and use them right in the `.onText` calls like

```js
const commands = { 
  START: /\/start/i,
  START_GAME: /начать/i,
  ...
} 
```

Use snake case (as in example) for this object

Reply:
```suggestion
const commands = {
  START: /\/start/i,
  START_GAME: /начать/i,
  STOP_GAME: /сдаюсь/i
};
```
fixed